app.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const path = require('path');
+const register = require('./api/registerRouter');
+const app = express();
+
+//body parser Midware
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.use('/static/login', express.static(path.join(__dirname, 'login')))
+app.use('/static/register', express.static(path.join(__dirname, 'register')))
+app.use('/static/homepage', express.static(path.join(__dirname, 'homepage')))
+app.use('/static/svg', express.static(path.join(__dirname, 'svg')))
+app.use('/static/logo', express.static(path.join(__dirname, 'logo')))
```

Comment:
Instead of doing it file by file, you can just server all the files from statics folder.
```js
app.use('/static', express.static(path.join(__dirname, 'public')))
```

But make sure you move these files to the "public" folder.